### ISSUE TITLE
23.04 Version

### ISSUE THREAD

**Issue by shawnhaywood** [mendhak/grub-reboot-picker#5]
Hello, tried installing on 23.04. Apparently not possible.  Any plans to create a version that will run on 23.04?  
Thanks!

**Comment by mendhak** [comment]
Hi I can't tell from your description what the issue is. Is the installation failing? Is the application running, or throwing an error? 

I just tried in Ubuntu 23.04 in a VM and it seems to work for me. 

![image](https://github.com/mendhak/grub-reboot-picker/assets/746276/b3694780-27b6-498d-847b-8b7c563761fb)

**Comment by mendhak** [comment]
Ah nope, scratch my last comment. My VM spun up a 22.04 instead of 23.04. I need to try 23.04. 

If you have any details of errors that will be useful.

**Comment by mendhak** [comment]
I understand now, there is no PPA for Ubuntu 23.04. 

I have made a new version and pushed it to Launchpad: https://launchpad.net/~mendhak/+archive/ubuntu/ppa/+packages

It should take a few hours to build and eventually appear.

**Comment by mendhak** [comment]
OK it's available now, give it a try. 

```bash
sudo add-apt-repository ppa:mendhak/ppa
sudo apt update
sudo apt install grub-reboot-picker
```

**Comment by shawnhaywood** [comment]
Works great now. Thanks!  Sorry for the lack of details.

**Comment by truonghm** [comment]
@mendhak hi, do you have any plans to make it available for 24.04 as well? I'm using Ubuntu 24.04 and currently running `sudo add-apt-repository ppa:mendhak/ppa` will complain that the repository does not have a release file. Hope this is added soon, and thank you for the great tool!

**Comment by mendhak** [comment]
@truonghm #6
